docs/apps/fingerprinting-demo.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 40
+title: Fingerprinting Demo
+---
+
+This demo app implements the PPhotonIQ Digital Fingerprinting to track site users. It identifies anonymous site visits, even if the use incognito or clear browsing data.
```

Comment:
Double P...

Reply:
"the PhotonIQ Digital Fingerprinting"
That seems strange wording.

Reply:
Good catches. Thanks!